tests/Behat/Content/Contact/GeneralContactContext.php

```diff
@@ -45,10 +46,14 @@ public function alterGeneralContactFields(BeforeParseEntityFieldsScope $scope):
     ];
 
     foreach ($scope->getFields() as $key => $value) {
+      $field_config = NULL;
+      if (isset($mapping[$key])) {
+        $field_config = FieldConfig::loadByName($scope->getEntityType(), $scope->getBundle(), $mapping[$key]);
```

Comment:
Let's move that into the method, I don't see any reason to replicate this every time.